### ISSUE TITLE
Track Evaluation

### ISSUE THREAD

**Issue by GeorgeHu6** [CAMMA-public/cholectrack20#7]
Amazing work! Cholectrack20 contains many precise annotations with very fine granularity. However, I noticed that there are no additional scripts in the `TrackEval` folder for evaluating all the models listed on the leaderboard using the Cholectrack20 dataset. Could you please clarify how to reproduce the results shown on the leaderboard?

**Comment by nwoyecid** [comment]
Listed models are public third-party SOTA models, you can reuse their default hyperparameters to finetune them on CholecTrack20 dataset. We have provided scripts that format CholecTrack20 to their individual data-loading format. When you have the finetuned models. We will update the `Trackeval` on this repo to support evaluating your model on this data to reproduce the leaderboard results by month end. The git pull request to do this has been pending for 11 months now.

**Comment by nwoyecid** [comment]
If you can't wait till month end, please visit the original metric repo: https://github.com/JonathonLuiten/TrackEval, look at pull requests, find the one related to CholecTrack20 and you would see the additional 2 scripts that you need + documentation. Thank you.

**Comment by GeorgeHu6** [comment]
Thank you for your reply! That helps me a lot.
